### PR TITLE
Fixup StackReference Architecture example

### PR DESCRIPTION
**Pull request by stack72**
Fixes: #544

### CODE REVIEW

**Comment by stack72** [comment]
Build fails due to:

`Error: Cannot find module 'js-yaml'`

File: aws-stackreference-architecture/networking/test.json [hunk 1]
```diff
@@ -0,0 +1,951 @@
+{
```
Comment (mikhailshilkov) [inline aws-stackreference-architecture/networking/test.json]: Is this file needed?

Comment (stack72) [reply aws-stackreference-architecture/networking/test.json]: It's removed :) Good catch!
